tests/test_tests.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from app import split_integer
+
+
+def test_could_split_on_different_parts(monkeypatch):
+    def split_on_equal_part(value: int, number_of_parts: int):
+        return [value // number_of_parts] * number_of_parts
+
+    monkeypatch.setattr(split_integer, "split_integer", split_on_equal_part)
+
+    test_result = pytest.main(["../app/test_split_integer.py"])
+    print(test_result)
+    assert test_result.value == 1
```

Comment:
I'd add here message. Because it is not obvious:
```suggestion
    assert test_result.value == 1, "This solution `split_on_equal_part` should not pass all tests"
```